packages/docs/docs/guides/9-request storage subgraph/0-entities-storage.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Storage Subgraph Entities
+keywords: [Subgraph, Entities, Schema]
+description: Subgraph entities and descriptions.
+---
+
+# Transaction
+
+Description: get specific details of the transaction
+
+| Field                | Type      | Description                                                                                  |
+| -------------------- | --------- | -------------------------------------------------------------------------------------------- |
+| id                   | ID!       | a unique ID based on IPFS hash & position of the transaction in the document                 |
+| hash                 | String!   | IPFS has                                                                                     |
+| channelId            | String!   | invoice ID for transaction                                                                   |
+| data                 | String    | request data is the current state of a request,                                              |
+| encryptedData        | String    | If the request is encrypted, the transactions are decrypted in this layer                    |
+| encryptionMethod     | String    | Advanced Encryption Standard (AES) or Elliptic Curve Integrated Encryption Scheme (ECIES)    |
```

Comment:
Please change to "Encryption methods for channel key and transactions, concatenated together"

Reply:
changed line 18